git-hooks/pre-commit

```diff
@@ -1,4 +1,7 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
Unless this is absolutely necessary, I prefer dash.
this is going to run on the CI server (albeit as Jenkins user), and I don't trust bash security records.

Reply:
It's to be able to use pipefail (I had resolved to that in pre-push).

It started without it, maybe I can try again: it should just mean removing the log on stdout and report differently, which might be a good thing. I am going to try that and update the PR.